DiegoFlores/SpringCRUD/src/main/java/com/bootcamp/vetPatitas/exception/InvalidDataException.java

```diff
@@ -0,0 +1,11 @@
+package com.bootcamp.vetPatitas.exception;
+
+public class InvalidDataException  extends NullPointerException {
```

Comment:
extends from NullPointerException???

you throw this Exception when input data is not correct... not when is null... right??